lib/challenge-alternative-solution-2.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const ATCQueue = function () {
+  this.aircraftQueue = []
+}
+
+ATCQueue.prototype.aircraftCount = function () {
+  return this.aircraftQueue.length
+}
+
+ATCQueue.prototype.enqueue = function (aircraft) {
+  this.aircraftQueue.push(aircraft)
+}
+
+ATCQueue.prototype.dequeue = function () {
+  const priorities = {
+    passenger: { large: 4, small: 3 },
+    cargo: { large: 2, small: 1 }
```

Comment:
This seems like a better solution than the original... it might be worth just replacing it?